src/main/java/com/pusher/client/channel/EventMetadata.java

```diff
@@ -0,0 +1,15 @@
+package com.pusher.client.channel;
+
+public class EventMetadata {
+    String userId;
+
+    public String getUserId() {
```

Comment:
I've implemented `getUserId()` and `setUserId()` as I thought it's more in the Java context. I can easily change it to 
```
public class EventMetadata {
    public String userId;
}
```
and then we can use it as `metadata.user_id`